app/models/bot/alegre.rb

```diff
@@ -20,6 +19,7 @@ def self.run(body)
         self.send_to_image_similarity_index(pm)
         self.send_to_title_similarity_index(pm)
```

Comment:
Need to send the description to the index too. I see we're still only sending the title, although you do search for title and description.

Reply:
lol fair point